conf/routes

```diff
@@ -19,57 +19,58 @@
 # ~~~~
 
 # Application calls
-GET        /                                    controllers.Application.dashboard()
-GET        /help                                controllers.Application.oldHelp()
-GET        /search                              controllers.Application.search()
-GET        /compare                             controllers.Application.compare()
-GET        /flowhistory                         controllers.Application.oldFlowHistory()
-GET        /jobhistory                          controllers.Application.oldJobHistory()
+GET        /                                        controllers.Application.dashboard()
+GET        /help                                    controllers.Application.oldHelp()
```

Comment:
Why indentation changes ?

Reply:
The name for the new REST endpoint was large to add without changing current indentation for better readability.